### ISSUE TITLE
New name

### ISSUE THREAD

**Issue by stanstrup** [duncan-ng/onto_shiny#1]
Probably we should come up with a better name than onto_shiny...

Brain-storming:

* Ontologyzer
* ZOOMAter
* ZOOMA en masse annotator (ZEMA)
* ZOOMA en masse shiny annotator (ZEMSA)
* User-Supervised Ontology Curator (U-SOC) 😆

**Comment by dygdedesener** [comment]
Good idea! I would pick Ontologyzer since we will probably use OLS to get annotation so picking up a name derived from ZOOMA should not make sense.

**Comment by duncan-ng** [comment]
Jan suggested a great name with U-SOC (User supervised ontology curator).
